imageio/v2.py

```diff
@@ -172,6 +179,13 @@ def __init__(self, plugin_instance: PluginV3, **kwargs):
         self.last_index = 0
         self.closed = False
 
+        if (
+            type(self.instance).__name__ == "PillowPlugin"
+            and kwargs.get("pilmode") is not None
```

Comment:
```suggestion
            and "pilmode" in kwargs
```

We still need to wait one more month for python 3.7 to reach EoL before we can remove 3.7 support and start using the walrus operator here. 

Until then, it should be enough to just check if the argument has been used without inspecting the value (I think).